production/mxnet_18.10-py3/Dockerfile

```diff
@@ -65,5 +69,6 @@ EXPOSE 22
 EXPOSE 8888
 
 
-ADD supervisord.conf /etc/supervisor/conf.d/supervisord.conf 
+ADD supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+RUN sed -i '/#!\/usr\/bin\/python/c\#!/usr/bin/python2' /usr/bin/supervisord
```

Comment:
This fix supervisord startup error after the system wide python to default python3.